cms/templates/base.html

```diff
@@ -56,11 +56,15 @@
 
     <a class="nav-skip" href="#main">${_("Skip to main content")}</a>
 
+    <%static:js group='base_vendor'/>
+
     <script type="text/javascript">
       window.baseUrl = "${settings.STATIC_URL | n, js_escaped_string}";
-      var require = {baseUrl: window.baseUrl};
+      require.config({
```

Comment:
Yes, that's how I thought of it. LMS does essentially the same thing, except it has to deal with the namespace. Here's what I copied (from https://github.com/edx/edx-platform/blob/master/lms/templates/main.html#L81):

```
  <script>
    window.baseUrl = "${settings.STATIC_URL}";
    (function (require) {
      require.config({
          baseUrl: window.baseUrl
      });
    }).call(this, require || RequireJS.require);
  </script>
```